Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM ubuntu:latest
+FROM --platform=linux/amd64 ubuntu:latest
 MAINTAINER Chris Collins <collins.christopher@gmail.com>
 
-ENV PKGS git x11vnc wget python python-numpy unzip xvfb curl tar bzip2 runit
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=America/New_York
```

Comment:
prevent stalling on prompt: `Please select the geographic area in which you live.`

https://stackoverflow.com/questions/61388002/how-to-avoid-question-during-the-docker-build